backend/all/all.go

```diff
@@ -18,6 +18,7 @@ import (
 	_ "github.com/rclone/rclone/backend/ftp"
 	_ "github.com/rclone/rclone/backend/googlecloudstorage"
 	_ "github.com/rclone/rclone/backend/googlephotos"
+	_ "github.com/rclone/rclone/backend/hdfs"
```

Comment:
Author of upstream library refused to fix the build for plan9 - https://github.com/colinmarc/hdfs/pull/256#issuecomment-751738114.
I suggest to move this line into a new `all/all_but_plan9.go` with stanza `// +build !plan9`

Reply:
But the below fix suggested by @ncw looks better

> like we do in the [sftp backend](https://github.com/rclone/rclone/blob/f7fe1d766b686078fb429710852a0cf6bdd08179/backend/sftp/sftp.go#L3) note that if we do that then we'll need an [hdfs_unsupported.go](https://github.com/rclone/rclone/blob/master/backend/sftp/sftp_unsupported.go) also.